react/src/components/molecules/EmergencyAlert/style.scss

```
@@ -0,0 +1,43 @@
+@import "global";
+@import '00-base/mixins';
+@import '02-molecules/emergency-alert';
+
+@mixin ma-emergency-alert ($color) {
+  color: $c-font-base;
+  background-color: tint($color, 90%);
+  border-color: tint($color, 50%);
```

I think in patternlab (shared scss) only the subsequent border top is set 
```suggestion
    & + & {
    border-top-color: $color;
  }
```